### PR TITLE
Feat/fa locale

### PR DESCRIPTION
**Pull request by masihjahangiri**

### CODE REVIEW

**Comment by mastermunj** [comment]
Hi @masihjahangiri, a big thank you for raising the PR.

Please help take care of following points:

* Use the format of language code and country code for naming the locale file. e.g. `en-US`
* Add unit tests.
* Use npm instead of yarn

**Comment by masihjahangiri** [comment]
Thanks @mastermunj. I fixed your points. please review again.

**Comment by masihjahangiri** [comment]
@mastermunj Thanks for make this elegant library. May you review my MR to add Fa language to your package.

**Review by mastermunj — CHANGES_REQUESTED** [review]

File: dist/to-words.js [hunk 1]
```diff
@@ -16,18 +16,24 @@ class ToWords {
...
+        /* eslint-disable @typescript-eslint/no-var-requires */
+        switch (this.options.localeCode) {
+            case 'en-IN':
+                return require('./locales/en-IN').Locale;
+            case 'en-MU':
+                return require('./locales/en-MU').Locale;
+            case 'en-US':
+                return require('./locales/en-US').Locale;
+            case 'fa':
+                return require('./locales/fa').Locale;
```
[the remark this answers is not in the document]
Comment (masihjahangiri) [reply dist/to-words.js]: @mastermunj Thanks for your notice, I updated it and built a repository.

**Review by mastermunj — APPROVED** [review]

**Comment by mastermunj** [comment]
Hi @masihjahangiri, I am making changes in code to include several of Indian languages, however, I am not having challenge understanding the code you wrote for `fa-IR` locale.

I need your help in understanding the change you made and how the number system there works so that it becomes easier for me to navigate through the code for making the changes I need to make.

**Comment by masihjahangiri** [comment]
Hi, @mastermunj sure. How can I help you?

**Comment by mastermunj** [comment]
I'd appreciate if you can elaborate the changes done [here](https://github.com/mastermunj/to-words/pull/216/files?file-filters%5B%5D=.json&file-filters%5B%5D=.ts&file-filters%5B%5D=dotfile#diff-a187ddcd6925a702834b3287af6b121257823ba0b154834afb93c86e185c20c6R87-R186).

Also, the logic behind adding options and decimalLengthWordMapping [here](https://github.com/mastermunj/to-words/pull/216/files?file-filters%5B%5D=.json&file-filters%5B%5D=.ts&file-filters%5B%5D=dotfile#diff-138f0d4c704276147ddb7802c044f245faf979f38e3c49fb09ddaa468091dca0R22-R38).

**Pull request merged**